nemoguardrails/colang/v2_x/library/avatars.co

```diff
@@ -313,29 +313,29 @@ flow tracking user talking state
   await user said something
   $user_talking_state = False
 
+@loop("state_tracking")
 flow tracking unhandled user intent state
-  # meta: loop_id=state_tracking
   global $user_intent_state
   when unhandled user intent as $flow
     $unhandled_user_intent_state = $flow.context.flow_name
-  orwhen FlowFinished(flow_id=r"^user ") as $event
+  orwhen FlowFinished(flow_id=regex("^user ")) as $event
     $unhandled_user_intent_state = None
 
+@loop("state_tracking")
 flow tracking visual choice selection state
-  # meta: loop_id=state_tracking
   global $choice_selection_state
   when VisualChoiceSceneAction.Started()
     $choice_selection_state = None
   orwhen VisualChoiceSceneAction.ChoiceUpdated() as $event
-    if "{{len($event.current_choice) > 0}}" == "True"
+    if "{len($event.current_choice) > 0}" == "True"
```

Comment:
You can now simplify even further
```
if len($event.current_choice) > 0
```

Reply:
Yes great! I pushed that with the new commit